src/main/grammars/RustParser.bnf

```diff
@@ -308,7 +309,7 @@ ExternAbi ::= extern STRING_LITERAL?
 
 RetType ::= '->' Type { pin = 1 }
 
-fake Function ::=
+Function ::=
   OuterAttr* default_? Vis? FnAttrs FnHeader FnParameters FnTail (';' | InnerAttrsAndBlock)
```

Comment:
I like the `Function` more: `Item` suffix does not much information. The whole `Item` concept is too fuzzy because of extern blocks, use declarations and anonymous impls.